echo-notifications/src/main/groovy/com/netflix/spinnaker/echo/notification/GithubNotificationAgent.java

```diff
@@ -128,14 +127,8 @@ public void sendNotifications(
   }
 
   private String getBranchCommit(String repo, String sha) {
-    Response response = githubService.getCommit("token " + token, repo, sha);
-    ObjectMapper objectMapper = EchoObjectMapper.getInstance();
-    GithubCommit message = null;
-    try {
-      message = objectMapper.readValue(response.getBody().in(), GithubCommit.class);
-    } catch (IOException e) {
-      return sha;
-    }
+    GithubCommit message =
+        Retrofit2SyncCall.execute(githubService.getCommit("token " + token, repo, sha));
```

Comment:
With the change in return type to GithubCommit from Response, retrofit2 is now doing the conversion, which is great.  Is there some equivalent try/catch block to add so this method returns the sha if the conversion fails?